.github/workflows/_static-analysis.yml

```diff
@@ -13,23 +13,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  version-check:
-    runs-on: macos-14
-    steps:
-      - uses: actions/checkout@v4
-      - name: Check version is up to date
-        run: |
-          make -f scripts/Makefile version
```

Comment:
Is this called somewhere else too, or is `make version` dead code now?

Reply:
Okay. That's in the release workflow or something?

Reply:
Nah we have to run it outside of CI because it modifies files. Post-GA, it'll be included in "release PRs" that also update relevant docs, like this one: #3988